minichain/p2p.py

```
@@ -207,6 +207,37 @@ def _validate_block_payload(self, payload):
+        if not isinstance(payload, dict):
+            return False
+        if set(payload) != {"from_height", "to_height"}:
+            return False
+        fh, th = payload.get("from_height"), payload.get("to_height")
+        if not isinstance(fh, int) or not isinstance(th, int):
+            return False
+        if fh < 0 or fh > th:
+            return False
+        return True
+
+    def _validate_blocks_payload(self, payload):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_validate_blocks_payload`

(ANN202)